### PR TITLE
fix naming in nessie client merge operation

### PR DESCRIPTION
**Pull request by rymurr**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/1357)

### CODE REVIEW

File: python/pynessie/nessie_client.py [hunk 1]
```diff
@@ -164,14 +164,14 @@ def assign_tag(self: "NessieClient", tag: str, to_ref: str, old_hash: Optional[s
         tag_json = ReferenceSchema().dumps(Tag(tag, self.get_reference(to_ref).hash_))
         assign_tag(self._base_url, tag, tag_json, old_hash, self._ssl_verify)
 
-    def merge(self: "NessieClient", branch: str, to_branch: str, old_hash: Optional[str] = None) -> None:
+    def merge(self: "NessieClient", branch: str, from_branch: str, old_hash: Optional[str] = None) -> None:
```
Comment (nastra) [inline python/pynessie/nessie_client.py:167]: should this be named `onto_branch` maybe? It seems we're using `onto` at a bunch of other code places as well

Comment (rymurr) [reply python/pynessie/nessie_client.py]: agreed, fixed

File: python/pynessie/nessie_client.py [hunk 2]
```diff
@@ -164,15 +164,15 @@ def assign_tag(self: "NessieClient", tag: str, to_ref: str, old_hash: Optional[s
         tag_json = ReferenceSchema().dumps(Tag(tag, self.get_reference(to_ref).hash_))
         assign_tag(self._base_url, tag, tag_json, old_hash, self._ssl_verify)
 
-    def merge(self: "NessieClient", branch: str, to_branch: str, old_hash: Optional[str] = None) -> None:
+    def merge(self: "NessieClient", onto_branch: str, from_branch: str, old_hash: Optional[str] = None) -> None:
```
Comment (snazy) [inline python/pynessie/nessie_client.py:167]: Can you adopt the order of "from" and "onto" to the same as in cli.py (or vice versa)?

Comment (rymurr) [reply python/pynessie/nessie_client.py]: done

File: python/pynessie/cli.py [hunk 3]
```diff
@@ -316,8 +316,8 @@ def tag(ctx: ContextObject, list: bool, force: bool, delete: bool, tag_name: str
 
 
 @cli.command("merge")
-@click.option("-b", "--branch", help="branch to merge onto. If not supplied the default branch from config is used")
-@click.argument("merge_branch", nargs=1, required=False)
+@click.option("-b", "--branch", "from_branch", help="branch to merge onto. If not supplied the default branch from config is used")
```
Comment (nastra) [inline python/pynessie/cli.py:319]: isn't this rather the `onto_branch` here? At least that's what the `help` description suggests.

It seems `from_branch` always defaults to `ctx.nessie.get_default_branch()` as there is no `--from-branch` flag (unless I'm missing something?

**Comment by rymurr** [comment]
Yeah, hence why the tests are failing still. Will update it soon....one of
those mornings :-P

**Comment by rymurr** [comment]
this should be ok now.

**Review by nastra — APPROVED** [review]

**Pull request merged**